src/ingest-pipeline/airflow/dags/utils.py

```diff
@@ -1111,9 +1111,10 @@ def make_send_status_msg_function(
     """
     def send_status_msg(**kwargs) -> bool:
         retcodes = [
-            int(kwargs['ti'].xcom_pull(task_ids=op))
+            kwargs['ti'].xcom_pull(task_ids=op)
             for op in retcode_ops
         ]
+        retcodes = [int(rc or '0') for rc in retcodes]
```

Comment:
Repeating an earlier question, why not have `or '0'` on line 1114 inside the `int()`?

Reply:
That would work- the 'or' clause in the list completion just seemed awkward to me.  Your approach is equivalent, and may be clearer.

Reply:
Thank you for your answers!